liblava/block/descriptor.hpp

```diff
@@ -110,7 +110,7 @@ struct descriptor : entity {
          * @return true     Create was successful
          * @return false    Create failed
          */
-        bool create(device_ptr device, VkDescriptorPoolSizesRef sizes, ui32 max = 1,
+        bool create(device_ptr device, VkDescriptorPoolSizesRef sizes, ui32 max = 0,
```

Comment:
for this case I would rather introduce a second `create` method where `max` is removed from the parameters